src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_VALUE = 11;
+    private ColorSupplier randomColor = new ColorSupplier();
+    private Random random = new Random();
+    private String white = Color.WHITE.toString();

```

Comment:
Please, consider better naming for `white` variable. Also, please avoid using `toString()` with enums. Instead, use `name() `method.